app/views/templates/ssh/module_action.erb

```diff
@@ -4,7 +4,7 @@ name: Module Action - SSH Default
 model: JobTemplate
 job_category: Modules
 description_format: "Module %{action} %{module_spec}"
-provider_type: SSH
+provider_type: Ansible
```

Comment:
@parthaa This is synced from community templates - looks like this change snuck in some how, can you update the community-template and I'll resync?

Reply:
Great job @stbenjam! I would merge already if there was not this issue.

Reply:
@stbenjam thanks!

https://github.com/theforeman/community-templates/pull/518